### PR TITLE
CIELAB terminology in colorzone

### PR DESCRIPTION
**Pull request by HSUfineprint**
If my understanding is correct, colorzone is using CIELAB. Then we should use the terminologies defined in CIE1976 L*a*b*. `lightness` for L*, and `chroma` instead of saturation

### CODE REVIEW

**Review by TurboGit — APPROVED** [review]
Looks good to me, thanks!

**Comment by elstoc** [comment]
Please can we watch out for these small changes, as any change to labels (though not tooltips) will have a documentation impact. I only spotted this because someone mentioned it on pixls.

**Comment by elstoc** [comment]
Sorry that was more for @TurboGit to remember to add the documentation-pending label to pull requests before merging.

**Comment by TurboGit** [comment]
Sorry @elstoc I forgot this one... I'm just human :)

**Comment by elstoc** [comment]
Is there a reason we only changed the names of the notebook pages, and not the names in the "select by" drop-down (which still says lightness, saturation, hue)?

**Comment by HSUfineprint** [comment]
No, it's a fault on my side. PR on the way

**Comment by elstoc** [comment]
shame because it leaves the module looking a bit half done for 4.4 (since we've now hit string freeze)

**Comment by HSUfineprint** [comment]
Sorry for my oversight. But changing saturation to chroma (in colorzone drop-down) should not cause any string issues right? Since these two words are used in other modules too, and be translated in all .po files.

**Comment by elstoc** [comment]
maybe, I guess that's up to @TurboGit

**Pull request merged**
